examples/chat-rn/src/App.tsx

```diff
@@ -75,7 +69,9 @@ function App() {
                     </Stack.Navigator>
                 </NavigationContainer>
             </Jazz.Provider>
-            <DemoAuthBasicUI appName="Jazz Chat" state={state} />
+            {state.state !== "signedIn" ? (
+                <DemoAuthBasicUI appName="Jazz Chat" state={state} />
+            ) : null}
```

Comment:
Conditional rendering of the auth UI is handled inside `DemoAuthBasicUI`

Reply:
it was still rendering some empty view - didn't look to deep into this, maybe a  bug in the auth implementation